controllers/topics.controller.js

```diff
@@ -6,5 +6,8 @@ exports.getTopics = (req, res, next) => {
         .then((topics) => {
             res.status(200).send({ topics });
         })
-        .catch(next);
+        .catch((err) => {
+            res.status(err.status).send({ error: err.message });
```

Comment:
👍 👍 for catching errors in the controller! Better to start setting up error handling middleware going forwards like we had a chat about the other day - I promise it's wroth spending a little time refactoring as it's going to make things so much easier in terms of testing/debugging, and make your code DRY and consistent too. Take a look at the notes here especially the last section for how to do that (next will pass errors to the middleware which is v cool https://notes.northcoders.com/courses/js-back-end/error-handling)